### PR TITLE
github(both): improve README.md and simplify PR templates

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tiflow/issues/3194

### What is changed and how it works?
- improve README.md, TiCDC currently does not support `cdc-log`
- The PR template is too complex and difficult for people to fill in, so simplify it.

### Check List 

Tests 

 - No code

### Release note 

```release-note
None
```

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!--
-Thank you for contributing to TiDB-CDC! Please read MD's [CONTRIBUTING](https://github.com/pingcap/tidb-cdc/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
+Thank you for contributing to TiFlow! 
+Please read MD's [CONTRIBUTING](https://github.com/pingcap/tidb-cdc/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
```
Comment (overvenus) [inline .github/pull_request_template.md:3]: It's no longer tidb-cdc.

**Review by nongfushanquan — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -8,7 +8,13 @@
...
 ## Introduction
 
-**TiFlow** is a unified data replication platform around [TiDB](https://docs.pingcap.com/tidb/stable), including two main components: DM and TiCDC. DM supports full data migration and incremental data replication from MySQL/MariaDB into [TiDB](https://docs.pingcap.com/tidb/stable). TiCDC supports replicating change data to various downstreams, including MySQL protocol-compatible databases, message queues via the open TiCDC protocol and other systems such as local file storage. More details can be found in [DM README](./README_DM.md) and [TiCDC README](./README_TiCDC.md).
+**TiFlow** is a unified data replication platform around [TiDB](https://docs.pingcap.com/tidb/stable),
+including two main components: DM and TiCDC.
+DM supports full data migration and incremental data replication from MySQL/MariaDB
+into [TiDB](https://docs.pingcap.com/tidb/stable).
+TiCDC supports replicating change data to various downstreams, including MySQL protocol-compatible databases
+and [Kafka](https://kafka.apache.org/).
+More details can be found in [DM README](./README_DM.md) and [TiCDC README](./README_TiCDC.md).
```
Comment (overvenus) [inline README.md:17]:
```suggestion
**TiFlow** is a unified data replication platform around [TiDB](https://docs.pingcap.com/tidb/stable),
including two main components:
* DM supports full data migration and incremental data replication from MySQL/MariaDB into [TiDB](https://docs.pingcap.com/tidb/stable).
* TiCDC supports replicating change data to various downstreams, including MySQL protocol-compatible databases and [Kafka](https://kafka.apache.org/).

More details can be found in [DM README](./README_DM.md) and [TiCDC README](./README_TiCDC.md).
```

File: README_TiCDC.md [hunk 3]
```diff
@@ -23,18 +25,18 @@ See a detailed introduction to [the TiCDC architecture](https://docs.pingcap.com
...
 
 When TiCDC is built successfully, you can find binary in the `bin` directory. Instructions for unit test and integration
 test can be found in [Running tests](./tests/integration_tests/README.md).
 
 ## Deployment
 
-You can setup a CDC cluster for replication test manually as following:
+You can set up a CDC cluster for replication test manually as following:
 
 1. Setup a TiDB cluster.
```
Comment (buchuitoudegou) [inline README_TiCDC.md:41]:
```suggestion
1. Set up a TiDB cluster.
```
same as above

File: README.md (hunk 2 above)
Comment (hi-rustin) [reply README.md]: Good suggestion!

File: .github/pull_request_template.md (hunk 1 above)
Comment (hi-rustin) [reply .github/pull_request_template.md]: Good catch!

**Review by buchuitoudegou — APPROVED** [review]

**Review by overvenus — APPROVED** [review]

**Pull request merged**